stock_vertical_lift_kardex/proxy/kardex-proxy.py

```diff
@@ -6,7 +6,7 @@
 import ssl
 import time
 
-import aiohttp
+import aiohttp  # pylint: disable=missing-manifest-dependency
```

Comment:
why not adding this dependency in the manifest instead of hiding this?

Reply:
because is not required. This is not for the module to work, this is need only if you run this proxy script.